src/Query/SemanticSearch/QuestionAnswering.php

```diff
@@ -7,7 +7,7 @@
 use LLPhant\Embeddings\Document;
 use LLPhant\Embeddings\EmbeddingGenerator\EmbeddingGeneratorInterface;
 use LLPhant\Embeddings\VectorStores\VectorStoreBase;
-use Symfony\Component\HttpFoundation\StreamedResponse;
+use Psr\Http\Message\StreamInterface;
 
 class QuestionAnswering
 {
```

Comment:
We should change the constructor as well so it can take an interface no?

Reply:
This is already done [here](https://github.com/ezimuel/LLPhant/blob/add/interface%2Bollama/src/Query/SemanticSearch/QuestionAnswering.php#L10)